langserve/api_handler.py

```diff
@@ -381,7 +381,14 @@ def _add_callbacks(
     config["callbacks"].extend(callbacks)
 
 
-class _APIHandler:
+_MODEL_REGISTRY = {}
```

Comment:
what is this used for?

Reply:
Even though I wrote the code I was wondering the same today :)

I'll try update the doc-string later -- it was needed to fix up some behavior associated with fast api / open api specs and having the same pydantic model appear multiple times causing the open api docs to crash, so instead this helps to look up an existing model if it was already seen (instead of creating a duplicate)